vendor/engines/bulk_email/app/controllers/bulk_email/bulk_email_controller.rb

```diff
@@ -15,17 +15,18 @@ class BulkEmailController < ApplicationController
 
     before_action :init_search_options
 
+    helper_method :datepicker_field_input
+    helper_method :user_type_selected?
+
     def search
-      if params[:search_type]
-        searcher = BulkEmailSearcher.new(@search_fields)
-        @users = searcher.do_search
-        @order_details = searcher.order_details
-      end
+      @searcher = BulkEmailSearcher.new(@search_fields)
+      @users = @searcher.do_search
```

Comment:
Just a note that nothing appeared to be using the `@order_details` results so I removed it. A lot of the tests focused on orders, but I've updated them to look at `@users` instead.